osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableTaikoHitObject.cs

```diff
@@ -1,26 +1,20 @@
 ﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
-using System.Collections.Generic;
 using osu.Framework.Graphics;
-using osu.Framework.Input;
+using osu.Framework.Input.Bindings;
 using osu.Game.Rulesets.Objects.Drawables;
 using osu.Game.Rulesets.Taiko.Judgements;
 using osu.Game.Rulesets.Taiko.Objects.Drawables.Pieces;
 using OpenTK;
-using OpenTK.Input;
 
 namespace osu.Game.Rulesets.Taiko.Objects.Drawables
 {
-    public abstract class DrawableTaikoHitObject<TaikoHitType> : DrawableScrollingHitObject<TaikoHitObject, TaikoJudgement>
+    public abstract class DrawableTaikoHitObject<TaikoHitType> :
+        DrawableScrollingHitObject<TaikoHitObject, TaikoJudgement>,
+        IKeyBindingHandler<TaikoAction>

```

Comment:
Can we move the colon to a new line and put the two lines above on a single line?
```
class DrawableHitObject
    : DrawableScrollingHitObject, IKeyBindingHandler
    where TaikoHitType : TaikoHitObject
```
Looks nicer I think